underdocs-renderer/src/main/kotlin/underdocs/renderer/parser/section/ReturnValueParse.kt

```diff
@@ -0,0 +1,76 @@
+package underdocs.renderer.parser.section
```

Comment:
Could you change the name of the file, to be more in line with other filenames? ReturnValueParse to ReturnValueParser?

Reply:
Oh thanks, nice catch! I almost left it this way. 
There is problem with my IDE recognizing this filevright after naming it to "...Parser". It's a mystery...

Reply:
I do hope, the issue vanishes by taking these measures!